README.md

```diff
@@ -16,11 +16,12 @@ in a company is among the best methods to attract and retain exceptional talent.
 In the United States,
 [since the 1950s](https://corpgov.law.harvard.edu/2014/10/02/what-has-happened-to-stock-options/),
 [stock options](#stock-options) have been a key part of pay for executives and other
-employees. As of **2014**, **7.2%** of all private sector employees (**8.5 million** Americans)
-and **13.1%** of *all* employees of companies with stock held stock options ([NCEO analysis](https://www.nceo.org/assets/pdf/articles/GSS-2014-data.pdf)
-of General Social Survey data).
+employees. In the United States in **2014**, **7.2%** of all private sector employees (**8.5
+million** people) and **13.1%** of *all* employees of companies with stock held stock options
+([NCEO analysis](https://www.nceo.org/assets/pdf/articles/GSS-2014-data.pdf) of General
```

Comment:
See Notion: Links for a note on the NCEO analysis. Is it fair to calculate the number (in millions) represented by 13.1% of the sample size?